components/web/php/php-8_2-ext-redis/Makefile

```diff
@@ -20,6 +20,7 @@ include ../../../../make-rules/shared-macros.mk
 
 COMPONENT_NAME=		redis
 COMPONENT_VERSION=	6.0.2
+COMPONENT_REVISION=	1
 COMPONENT_SRC=		$(COMPONENT_NAME)-$(COMPONENT_VERSION)
```

Comment:
There is no web/php-82/extension/php-redis yet so COMPONENT_REVISION is bogus here.